mxcube3/routes/SampleCentring.py

```diff
@@ -428,6 +428,36 @@ def delete_shape(sid):
                "Zoom 6", "Zoom 7", "Zoom 8", "Zoom 9", "Zoom 10"]
 
 
+@mxcube.route("/mxcube/api/v0.1/shapes/rotate_to", methods=['POST'])
```

Comment:
Juts to understand, why is this needed? Is it not enough with move to point?

Reply:
Its to be able to rotate back to the angle where a grid was defined but not to move to a certain
position